apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -80,7 +86,7 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
         )
     })
 
-    test('insert api should fail if item already exists', async() => {
+    testIfSupportedOperationsNotIncludes(supportedOperations, [continueInsertingOnError])('insert api should fail if item already exists', async() => {
```

Comment:
REALLY confusing, create another operation that every implementation supports but mongo- 
`AtomicBulkInsert` and IMO remove this new `testIfSupportedOperationsNotIncludes`